app/AccountancyModule/PaymentModule/components/PaymentDialog.php

```diff
@@ -115,7 +116,12 @@ protected function createComponentForm(): BaseForm
             $group = $this->paymentService->getGroup($this->groupId);
             Assertion::notNull($group);
 
-            $nextVS = $this->paymentService->getNextVS($this->groupId);
+            try {
+                $nextVS = $this->paymentService->getNextVS($this->groupId);
+            } catch (InvalidVariableSymbol $exception) {
+                $this->flashMessage('Nelze vygenerovat následující VS: \'' . $exception->getMessage() . '\'', 'danger');
```

Comment:
Tohle bych nědělal jako flash message. Protože ta flash message se zobrazí až po redirectu, což je spíš matoucí.

Asi bych to nahradil tím prázdným stringem.

Reply:
Nechapu jak to myslis, jako cim bys to nahradil

Reply:
Nahradil bych to nějakou proměnnou předávanou do šablony a zobrazoval u toho textového pole, kde se to převyplňuje. Takhle to nebude vidět, protože

a) flash messages v té komponentě nikde nezobrazujeme
b) flash messages se zobrazí až po redirectu, redrawu.

Reply:
pokud je už nyní posední VS na hranici (9999999999) tak se rovnou zobrazuje. 

a když to není na hranici ale dojde na to číslo, tak to taky vypíše, 

tedy nevidim nikde problém